### DISCUSSION TITLE
MacOS运行程序时报错

### DISCUSSION THREAD

**Discussion by FallenWind2531** [mikumifa/biliTickerBuy#451] [提问专区]
<img width="1440" alt="WechatIMG57" src="https://github.com/user-attachments/assets/1633d2ee-00fb-448c-ae09-5702c88e4f34">
这是什么环境没配好吗

**Comment by mikumifa** [comment]
具体系统 ？

**Reply by FallenWind2531** [reply]
macOS版本12.0.1，M1的芯片

**Reply by mikumifa** [reply]
试一下amd64那个

**Reply by FallenWind2531** [reply]
截图里试的就是v2.8.9的amd64那个，刚刚试了下patch3的amd64也遇到了这个问题

**Reply by NomadZhang** [reply]
hxd，我看到最下面的一行报错，numpy和python的版本信息对不上，考虑一下降低一下numpy的版本？

**Reply by FallenWind2531** [reply]
> hxd，我看到最下面的一行报错，numpy和python的版本信息对不上，考虑一下降低一下numpy的版本？

感谢，应该就是这个原因，不过我换到windows上用了，就不折腾版本了XD

**Comment by mikumifa** [comment]
先mark一下，我暂时还没有复现出这种问题，因为我手上的mac是inter的
